src/Particle/Walker.h

```diff
@@ -159,93 +160,90 @@ struct Walker
     Grad_GPU.resize(N);
     Lap_GPU.resize(N);
     // For GPU coallescing
-    k_species_stride = ((2*num_k + 15)/16) * 16;
+    k_species_stride = ((2 * num_k + 15) / 16) * 16;
     if (num_k)
-      Rhok_GPU.resize (num_species * k_species_stride);
-  } GPU_XRAY_TRACE
-  inline CUDA_PRECISION_FULL* get_rhok_ptr ()
-  {
-    return Rhok_GPU.data();
-  } GPU_XRAY_TRACE
-  inline CUDA_PRECISION_FULL* get_rhok_ptr (int isp)
-  {
-    return Rhok_GPU.data() + k_species_stride * isp;
-  } GPU_XRAY_TRACE
+      Rhok_GPU.resize(num_species * k_species_stride);
+  }
+  GPU_XRAY_TRACE
```

Comment:
It looks like GPU_XRAY_TRACE is placed at the end of the function definition?
If so, it's clearer in the original formatting.   Blank lines separating the functions would also work to make it clear which function GPU_XRAY_TRACE modifies (assuming the formatter would preserve the blank lines)